test/integration/status_test.go

```diff
@@ -40,7 +40,7 @@ func TestInsufficientStorage(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), Minutes(5))
 	defer Cleanup(t, profile, cancel)
 
-	startArgs := []string{"start", "-p", profile, "--output=json", "--wait=true"}
+	startArgs := []string{"start", "-p", profile, "--memory=1900", "--output=json", "--wait=true"}
```

Comment:
why the memory allocation here?

Reply:
so it doesn't auto select 6GB for a test that doesnt need it